### DISCUSSION TITLE
This user account is blocked.

### DISCUSSION THREAD

**Discussion by LeVanAnhDuc** [vercel/community#5076] [Account Issue] [answered]
### Question

When I register a vercel account with my github account:
- LeVanAnhDuc. 
Then I was told that my account was locked. I don't know what I did that led to the above situation. Looking forward to your earliest response.
My email: levananhduc1804@gmail.com.
![block account](https://github.com/vercel/community/assets/138034754/d3511923-eb3a-44f9-aa57-74952553a8ea)

**Comment by swarnava** [comment, marked as answer]
Hi,

You can email us at registration@vercel.com.

**Reply by DefenderConsultants** [reply]
Hi, I'm also getting this issue, do you know how long the response time is after emailing? Thanks
